experiment_diagrams/README.md

```diff
@@ -0,0 +1,37 @@
+# IM3 Experiment Diagrams
+
+Diagrams representing each experiment in the IM3 project.
+
+To update a diagram:
+* If you feel uncomfortable interacting with git, [open an issue](https://github.com/IMMM-SFA/im3components/issues/new?assignees=thurber&labels=documentation&template=update-experiment-diagram.md&title=Update+Experiment+Diagram) and a DSC team member will help you through the process.
+* Otherwise:
+    * Create a new branch from `develop`.
+    * Make your updates in that branch.
+    * Include an updated static image (or a DSC team member can add it to your pull request).
```

Comment:
potentially clarify if you mean to add the static image in the experiment dir or to the README